docker-compose.yml

```diff
@@ -147,7 +149,11 @@ services:
       disable: true
 
   superset-node:
-    image: node:18
+    build:
```

Comment:
This makes things a bit more DRY and in sync with the Dockerfile, main thing is the base is set in docker for this, and there's a `frontend-mem-nag` script that checks some assumption and raises message if docker conditions aren't set properly.

Reply:
That memory nag seems to be set to 3GB. Seems like we might as well raise it, no? Not sure what a reasonable minimum is at this point, but something significant - maybe 10GB?

Reply:
ok let's try 10, unclear what exactly is needed and different contexts, but 10 seems much more accurate than 3

Reply:
if people have ideas as to how to contain this docker beast it'd be great